geth/params/config.go

```diff
@@ -96,6 +96,10 @@ type WhisperConfig struct {
 	// Currently, it's used by MailServer.
 	PasswordFile string
 
+	// Password for symmetric encryption
+	// (if no account file selected, then this password is used for symmetric encryption).
```

Comment:
I would explain it a bit more. It's used only for MailServer so maybe something like `Password is used to authenticate with MailServer.`. It's not used for any other symmetric encryption at the moment.